client/dbus/mocks/DBusCallResponse.go

```diff
@@ -12,8 +12,6 @@
 //    See the License for the specific language governing permissions and
 //    limitations under the License.
 
-// Code generated by mockery v2.1.0. DO NOT EDIT.
-
```

Comment:
Have we edited this file manually? We'll run into conflicts if we want to use `mockery` at a later point.

Reply:
No, I have a different version of mockery and I have to restore the right comments, sorry.
Fixed.